scapy/fields.py

```diff
@@ -2155,3 +2155,38 @@ def i2repr(self, pkt, x):
         elif self.use_msec:
             x = x / 1e3
         return "%s sec" % x
+
+
+class ScalingField(Field):
```

Comment:
Could you have documentation about that field in here ? Thanks !

Reply:
Done. Is it sufficient?

Reply:
That’s great thanks !